### DISCUSSION TITLE
[Feature Request] Texture Pack support

### DISCUSSION THREAD

**Discussion by trololirko** [HarbourMasters/2ship2harkinian#527] [Ideas]
On SOH there is support for external HD texture packs. Are there plans to implement this into 2S2H? I didn't catch it on the roadmap. :)

**Comment by garrettjoecox** [comment]
Yes, out of the box 2S2H has the same level of support for texture packs. Our tooling (retro) needs a bit of work, but people have found workarounds and are already cooking texture packs up. Check the #2s2h-modding forum in discord
